src/modules/pageIcons/pageIcons.css

```diff
@@ -16,14 +16,11 @@
     margin-right: 0.34em;
     font-family: var(--icon-font);
     text-align: center;
-    align-self: end;
+    align-self: center;
     line-height: inherit;
 }
 .awLi-hideTitle .awLi-icon {
-    margin-right: 0;
-}
-.namespaces .awLi-hideTitle .awLi-icon {
```

Comment:
There are several places that need margin adjustment, not only hierarchy. Also linked references and namespace macro. It is easier to keep general css rule.

Also change the margin value from default to moree symmetric.
Before:
<img src="https://user-images.githubusercontent.com/1984175/204209965-b180e550-da4f-499b-97f4-7c0753f88c9a.png" width="80">

After:
<img src="https://user-images.githubusercontent.com/1984175/204210063-70e22e3d-1922-417d-9fd0-20bf2643b624.png" width="80">